tests/ingestion/test_light_rail_gps.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+from lamp_py.ingestion.light_rail_gps import raw_gps_schema
+from lamp_py.ingestion.light_rail_gps import dataframe_from_gz
+
+from ..test_resources import test_files_dir
+
+mock_file_list = [
+    os.path.join(
+        test_files_dir,
+        "INCOMING/2024-05-01T02:30:11Z_s3_mbta_ctd_trc_data_rtr_prod_LightRailRawGPS.json.gz",
+    )
+]
+
+
+def test_light_rail_gps() -> None:
+    """
+    test gtfs_events_for_date pipeline
+    """
+    dataframe, archive_files, error_files = dataframe_from_gz(mock_file_list)
```

Comment:
Not totally sure if it's worth testing the rest of the process, which just includes a `file_list_from_s3` call and the parquet writing.